### PR TITLE
Tweaks for system status syntax

### PR DESCRIPTION
**Pull request by shivapoudel**
:)

### CODE REVIEW

File: includes/admin/views/html-admin-page-status-report.php [hunk 1]
```diff
@@ -111,16 +109,15 @@
 				// Check if phpversion function exists
 				if ( function_exists( 'phpversion' ) ) {
 					$php_version = phpversion();
-
 					if ( version_compare( $php_version, '5.4', '<' ) ) {
-						echo '<mark class="error">' . sprintf( __( '%s - We recommend a minimum PHP version of 5.4. See: <a href="%s" target="_blank">How to update your PHP version</a>', 'woocommerce' ), esc_html( $php_version ), 'http://docs.woothemes.com/document/how-to-update-your-php-version/' ) . '</mark>';
+						echo '<mark class="error">' . sprintf( __( '%s - We recommend a minimum PHP version of 5.4. See: <a href="%s" target="_blank">How to update your PHP version</a>', 'woocommerce' ), esc_html( $php_version ), ' http://docs.woothemes.com/document/how-to-update-your-php-version/' ) . '</mark>';
```
Comment (SiR-DanieL) [inline includes/admin/views/html-admin-page-status-report.php]: Why did you add a space here?

Comment (shivapoudel) [inline includes/admin/views/html-admin-page-status-report.php]: Fixed up xD
